genshin/client/routes.py

```diff
@@ -158,3 +158,18 @@ def get_url(self, region: types.Region, game: types.Game) -> yarl.URL:
     bbs="https://webstatic-sea.mihoyo.com/admin/mi18n/bbs_cn/m11241040191111/m11241040191111-{lang}.json",
     inquiry="https://mi18n-os.hoyoverse.com/webstatic/admin/mi18n/hk4e_global/m02251421001311/m02251421001311-{lang}.json",
 )
+
+BBS_URL = InternationalRoute(
+    overseas="https://bbs-api-os.hoyolab.com/",
+    chinese="https://bbs-api.mihoyo.com/",
+)
+
+BBS_REFERER_URL = InternationalRoute(
+    overseas="https://www.hoyolab.com/",
+    chinese="https://bbs.mihoyo.com/",
+)
+
+BBS_GET_USER_URL = InternationalRoute(
+    overseas="/community/painter/wapi/user/full",
+    chinese="/user/wapi/getUserFullInfo",
+)
```

Comment:
Routes should ideally only be full urls. I recommend keeping this part in `get_hoyolab_user`

Reply:
Fixed.